### ISSUE TITLE
Interpreting training results and showing loss stats/graph

### ISSUE THREAD

**Issue by ghost** [ultralytics/yolov5#1468]
I'm new to yolov5 and I'm trying to build a single class object detector(detecting mouth in an image).
Here are the results I got:
![results](https://user-images.githubusercontent.com/74835824/99886088-67733800-2c42-11eb-9bff-9e909ab59842.png)
I'm pretty sure that my dataset is problematic and thus the training wasn't successful, but putting this aside, I have two questions:
1.  how do I interpret the results and graphs above? I used Keras before and the results were pretty easy to understand, as they included accuracy(which is irrelevant here as I understand) and loss metrics for each epoch.
2. I viewed several other projects in which the output includes GloU([example](https://user-images.githubusercontent.com/26833433/97808309-8182b180-1c66-11eb-8461-bffe1a79511d.png)) - why don't I have this graph in my results and what does "GloU" mean?

PS - I have also tried running train.py on a dataset that includes a single image of my mouth labeled - the same image is used for training, validation, and testing. I expected the model to be able to learn to label this image correctly as it is the only data it gets, but when it comes to testing or validation - the model isn't able to detect anything above 0.05 confident rate after 200 epochs - and even then it draws random boxes and doesn't detect the mouth correctly. Is it the expected result, or does it indicates that something is wrong in my setup?

**Comment by glenn-jocher** [comment]
GIoU term is deprecated: https://github.com/ultralytics/yolov5/pull/1120

Follow custom training tutorial:
https://docs.ultralytics.com/yolov5/tutorials/train_custom_data

**Comment by ghost** [comment]
> GIoU term is deprecated: #1120
> 
> Follow custom training tutorial:
> https://docs.ultralytics.com/yolov5/tutorials/train_custom_data

I have already followed the custom training tutorial, but I still don't know what each graph in the results.png says. Is there a loss graph? How should I measure the results? 
I read about precision, recall(I think that these graphs are irrelevant as I have only one class), and mAP- but I still don't understand what do mAP@0.5, box, and objectness mean.

**Comment by glenn-jocher** [comment]
@naveedri oh, for the metrics I would recommend the original YOLOv3 paper to brush up on the basics:
https://arxiv.org/abs/1804.02767

and wikipedia for metrics:
https://en.wikipedia.org/wiki/Precision_and_recall

**Comment by ghost** [comment]
> @naveedri oh, for the metrics I would recommend the original YOLOv3 paper to brush up on the basics:
> https://arxiv.org/abs/1804.02767
> 
> and wikipedia for metrics:
> https://en.wikipedia.org/wiki/Precision_and_recall

Thank you so much!
Can you also refer to this part?
> PS - I have also tried running train.py on a dataset that includes a single image of my mouth labeled - the same image is used for training, validation, and testing. I expected the model to be able to learn to label this image correctly as it is the only data it gets, but when it comes to testing or validation - the model isn't able to detect anything above 0.05 confident rate after 200 epochs - and even then it draws random boxes and doesn't detect the mouth correctly. Is it the expected result, or does it indicates that something is wrong in my setup?

Here are the results I got on this (again, single image, 200 epochs
![results](https://user-images.githubusercontent.com/74835824/99897114-7dfbac80-2c9f-11eb-8de4-b48f37e50466.png)

**Comment by glenn-jocher** [comment]
> I'm pretty sure that my dataset is problematic and thus the training wasn't successful

You require a valid dataset for valid results.
